use anyhow::Error;

#[actix::main]
async fn main() -> Result<(), Error> {
    println!("Hello, world!");
}
